LSpec/LSpec.lean

```diff
@@ -0,0 +1,55 @@
+import Lean
+
+abbrev Rel (α : Type _) := α → Bool 
```

Comment:
Would it be interesting to consider `α → Result` so as to not restrict it to `bool`?